<?xml version="1.0"?>
<launch>

  <node pkg="tf" type="static_transform_publisher" name="odom_broadcaster" args="0 0 0 0 0 0 map odom 10" />
  <node pkg="tf" type="static_transform_publisher" name="base_link_broadcaster" args="0 0 0.02 0 0 0 base_footprint base_link 10" />
  <node pkg="tf" type="static_transform_publisher" name="sonar_broadcaster" args="0.2 0 0 0 0 0 base_link sonar_range 10" />
  
  <!-- 
  <node pkg="m_car_base_control" type="teleop2.py" name="teleop_node" />
   -->
   
  <node name="serial_node"        pkg="rosserial_python"      type="serial_node.py">
    <param name="port"              type="string"               value="/dev/ttyACM0"/>
    <param name="baud"              type="int"                  value="57600"/>
  </node>
    
  <include file="$(find iahrs_driver)/launch/iahrs_driver.launch" />
  
 
  <param name="imu_offest_angle"    type="double"   value="30.0"/> 
  <node pkg="henes_car_control" type="Henes_Car_Control" name="henes_base_control_node" output="screen"/>
  
   
   <node pkg="henes_car_control" type="odom_path_node.py" name="path_node"/>
</launch>
    

